compliance/cis_k8s/rules/cis_5_1_5/rule.rego

```diff
@@ -0,0 +1,62 @@
+package compliance.cis_k8s.rules.cis_5_1_5
+
+import data.compliance.cis_k8s
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+import future.keywords.in
+
+# Ensure that default service accounts are not actively used (Manual)
+default rule_violation = false
+
+# no roles or cluster roles bound to default service account apart from the defaults.
```

Comment:
Yes, as I mentioned beforehand about this change, it's not by a mistake. 
I was thinking about a way to make our rules structure more concrete, i.e.:

```rego
finding = result {
    # filter

    # evaluate

    # set result
}
```
some rules calculate the `rule_evaluation` outside of the finding clause, which is something I try to avoid here (prevents unexpected behavior). by using de-morgen law you can see that instead of ANDing non_violations, we just need one violation to occur, this will let rego do the ORing for us and in turn will let us calculate the rule_evaluation inside the finding clause, and in one line which makes it easier to read (using `not` instead in the future).

This structure can be reproduced across our repo and will make our rules' structure more concrete, and will make sure we don't calculate the rule_evaluation outside of the findings clause.

Reply:
Lets discuss it